Libraries/LibLine/Editor.h

```diff
@@ -78,6 +81,7 @@ class Editor {
 
     Function<Vector<String>(const String&)> on_tab_complete_first_token = nullptr;
     Function<Vector<String>(const String&)> on_tab_complete_other_token = nullptr;
+    Function<void(Editor&)> on_display_refresh = nullptr;
```

Comment:
No need to explicitly initialize `Function` objects with `nullptr`, as they will default-initialize the same way.